Documentation/help_files/message_center_help.php

```diff
@@ -0,0 +1,439 @@
+                
+                <p><?php echo xlt("Additional functionality including the ability to send SMS text messages, automated phone dialing, emails etc. can be accessed by enabling the optional MedEx Communication Service");?>.
+                
+                <ul>
+                    <li><a href="#messages"><?php echo xlt("Messages");?></a></li>
+                    <li><a href="#reminders"><?php echo xlt("Reminders");?></a></li>
+                    <li><a href="#recalls"><?php echo xlt("Recalls");?></a></li>
+                    <li><a href="#medex_communication_service"><?php echo xlt("MedEx Communication Service");?></a></li>
+                    <li><a href="#medex_enabled_recall_board"><?php echo xlt("MedEx Enabled Recall Board");?></a></li>
+                    <li><a href="#sms_zone"><?php echo xlt("SMS zone - enabled only with the optional MedEx Communication Service");?></a></li>
+                </ul>
+            </div>
+            <div class= "row" id="messages">
+                <h4 class="oe-help-heading"><?php echo xlt("Messages"); ?><a href="#"><i class="fa fa-arrow-circle-up float-right oe-help-redirect" aria-hidden="true"></i></a></h4>
+                <p><?php echo xlt("Lets you send messages to staff members about patient related matters");?>.
+                
+                <p><i class="fa fa-exclamation-circle oe-text-orange"  aria-hidden="true"></i> <strong><?php echo xlt("It is a part of the medical record"); ?>.</strong>
+                
+                <p><?php echo xlt("Upon logging in to openEMR a small envelope icon is visible on the top right. It shows the number of pending messages and dated reminders"); ?>.
+                <button type="button" class="btn btn-default btn-sm oe-no-float"><i class="fa fa-envelope"></i></button>
+                
+                <p><?php echo xlt("You can click on the envelope icon to access the messaging center"); ?>.
+                
+                <p><?php echo xlt("Alternatively it can be accessed by clicking in the Messages menu item in the top navigation bar"); ?>.
+                
+                <p><?php echo xlt("The messaging center will open with the Messages tab activated and will display the logged in user's messages as indicated by the caption  My Messages "); ?>.
+                
+                <p><i class="fa fa-exclamation-circle oe-text-orange"  aria-hidden="true"></i> <strong><?php echo xlt("For those with administrative privileges an icon is displayed next to the My Messages caption") ?>.</strong>
+                
+                <p><?php echo xlt("Clicking on the icon next to the caption will display everyone's messages and the caption will change to  All Messages") ?>.
+                
+                <p><?php echo xlt("By default  Active Messages  are displayed as indicated in the section below the Messages caption"); ?>.
```

Comment:
Ensure no double spacing in these translation constants (such as the line above)

Reply:
Will do